backend/src/test/services/ProjectService.test.ts

```diff
@@ -101,7 +101,7 @@ describe('ProjectService tests', () => {
       },
     };
     const projectService = container.resolve(ProjectService);
-    await expect(projectService.createProject(PROJECT_INFO as ProjectRequestDTO)).rejects.toThrowError(
+    await expect(projectService.createProject(PROJECT_INFO as unknown as ProjectRequestDTO)).rejects.toThrowError(
```

Comment:
Why is the unknown needed here?

Reply:
it gave me a DTO error because I'm missing a variable. However, I remove that variable on purpose to make the test fails. But it still wants PROJECT_INFO to have the same exact value as ProjectRequestDTO

Reply:
I think something changed in the ProjectRequestDTO, I don't know when but I didn't change that file. I found that error after I recheck all the backend tests.

Reply:
Oh ok